label_studio/io_storages/base_models.py

```diff
@@ -688,12 +688,27 @@ class Meta:
 class ImportStorageLink(models.Model):
 
     task = models.OneToOneField('tasks.Task', on_delete=models.CASCADE, related_name='%(app_label)s_%(class)s')
+
+    # ---
+    # NOTE: ImportStorageLink is task-level, not key-level. These fields could be split out into a new, key-level table to optimize DB access patterns.
```

Comment:
We should remove this, it's too speculative. Happy to have this in a doc, but in code, I don't feel it's helpful.

```suggestion
```